docs/chapters/developer_guide.tex

```diff
@@ -0,0 +1,176 @@
+	teamcity-messages (1.32)      &                 \\ \hline
+\end{longtable}
+
+\section{Tooling}
+The tool is using:
+\begin{enumerate}
+\item \keyw{anaconda} / \keyw{miniconda} /\keyw{conda} to setup its python environment. 
+\item \keyw{poetry} to install the python packages for the environment
+\item \keyw{VSCode} to edit python files, run the created (unit)tests, see/visualize the test coverage
+\end{enumerate}
+
+\subsection{Utility / install scripts}
+We use several other batch scripts.
+\begin{itemize}
+	\item \keyw{CondaInstall.bat} used to install miniconda on the client pc, you need to restart the main script (DevelopDfastmi.bat) in a \textbf{new} command line prompt because the command line prompt environment is
+	 updated after install.
+	\item \keyw{VSCodeInstall.bat} used to install visual studio code on the client pc, you need to restart the main script (DevelopDfastmi.bat) in a \textbf{new} command line prompt because the command line prompt environment is updated after install.
+\end{itemize}
+
+\subsection{VSCode Extensions}
+In VSCode we use extensions to find python unit test, visualize coverage and debug our code. To do this we install the following extensions.
+\begin{longtable}{|l|l|}
+	\caption{Extensions for developing with VS Code.} \\
+	\hline
+	\textbf{VS Code extension} & \textbf{Remark} \\ \hline
+	\endfirsthead
+	%
+	\multicolumn{2}{c}%
+	{{\bfseries Extensions for developing with VS Code \thetable\ continued from previous page}} \\
+	\hline
+	\textbf{VS Code extension} & \textbf{Remark} \\ \hline
+	\endhead
+	%
+	Cameron.vscode-pytest                 		&                 \\ \hline
+	donjayamanne.python-environment-manager     &                 \\ \hline
+	hbenl.vscode-test-explorer                 	&                 \\ \hline
+	littlefoxteam.vscode-python-test-adapter    &                 \\ \hline
+	ms-python.pylint                 			&                 \\ \hline
+	ms-python.python                 			&                 \\ \hline
+	ms-python.vscode-pylance                 	&                 \\ \hline
+	ms-vscode.test-adapter-converter            &                 \\ \hline
+	ryanluker.vscode-coverage-gutters           &                 \\ \hline
+\end{longtable}
+
+
+\subsection{VSCode userfiles}
+The user files can be found in subfolder \textbf{.vscode}
+\begin{enumerate}
+	\item \keyw{extensions.json} will reference the expected and recommended visual studio code extensions. It will not install them but inform the user what is missing to start developing.
+	\item \keyw{launch.json} will configure VS Code what to do when an user will debug unit tests or start debugging the application.
+	\item \keyw{settings.json} configured settings for 
+\end{enumerate}
+
+\subsection{Python/TOML}
+This file is used by the python interpreter to configure the application. The TOML file itself has nothing to do with python, poetry or VS Code but it is a configuration file which is easy to read.
```

Comment:
This text does not seems correct and it seems in contradiction with what you stated earlier in chapter Configuration files.
The start of this text refers to a specific file, but which file is not fully clear, only that it is a toml file.